test/relativeUrlPrefixRule.spec.ts

```diff
@@ -105,6 +120,21 @@ describe(ruleName, () => {
           source
         });
       });
+
+        it('should fail when a relative URL is prefixed by ../', () => {
+            const source = `
+                    @Component({
+                        templateUrl: '.././foobar.html'
```

Comment:
The style guide Angular says: 
> Why? A component relative URL requires no change when you move the component files, **as long as the files stay together**.

I understood that like 'in the same folder' (mainly in an Angular CLI environment)  but, indeed, it can lead to different interpretations.

Our rationale is `when you move the component files`. So, we should accept all relative urls. 
In my IDE, if I move a component with a relative url, my IDE fixes the new path automatically. 
I agree that `./../` is unclean but, for me, it's the same thing than `../`.

Do you know what I mean ? I can fix it quickly with this in mind (we accept all relative paths).